### PR TITLE
Poolseparation

### PR DESCRIPTION
**Pull request by elduddz**
creates a Ubuntu pool and a WS pool separate from the agents pool

### CODE REVIEW

File: agentinstall.ps1 [hunk 1]
```diff
@@ -64,14 +64,25 @@ wget "https://vstsagentpackage.azureedge.net/agent/$agentVersion/$zip" -OutFile
...
-    $agentName = "$ComputerName-A$i"
+    $agentName = "$ComputerName-ws2019-A$i"
     
     Expand-Archive -Path ./$zip -DestinationPath ./$agentDir
     
     cd $agentDir
         
-    .\config.cmd --unattended --url https://dev.azure.com/$account --auth PAT --token $PAT --pool "$pool" --agent $agentName --runAsService
-    
+    .\config.cmd --unattended --url https://dev.azure.com/$account --auth PAT --token $PAT --pool "$pool Agents" --agent "$agentName" --runAsService
```
Comment (MarcDenman) [inline agentinstall.ps1]: I don't think pool is the correct variable name here, I read it and thought it was the full pool name. $PoolNamePrefix?

File: agentinstall.ps1 [hunk 2]
```diff
@@ -64,14 +64,25 @@ wget "https://vstsagentpackage.azureedge.net/agent/$agentVersion/$zip" -OutFile
...
+}
+
+for ($i = 1; $i -le $count; $i++) {
```
Comment (MarcDenman) [inline agentinstall.ps1:81]: Can you just leave a stating what this loop is doing differently to the other one . Very similar apart from the pool name.

File: agentinstall.ps1 (hunk 1 above)
Comment (elduddz) [reply agentinstall.ps1]: done

File: agentinstall.ps1 (hunk 2 above)
Comment (elduddz) [reply agentinstall.ps1:81]: done

**Review by MarcDenman — APPROVED** [review]

**Pull request merged**